src/pages/home/report/PureReportActionItem.tsx

```diff
@@ -899,7 +895,6 @@ function PureReportActionItem({
                             CONST.REPORT.ACTIONABLE_MENTION_INVITE_TO_SUBMIT_EXPENSE_CONFIRM_WHISPER.DONE,
```

Comment:
This does not have correct params passed `formatPhoneNumber` needs to be removed as well.